src/components/Verses/BookmarksAndQuickLinks.tsx

```diff
@@ -16,12 +14,9 @@ enum View {
 
 const BookmarksAndQuickLinks = () => {
   const { t } = useTranslation('home');
-  const [view, setView] = useState(View.Popular);
+  const [view, setView] = useState(View.Bookmarks);
```

Comment:
Since we are not using `popular` here anymore, it makes sense to rename the component + remove the `useState` and `{view === View.Bookmarks && <BookmarkedVersesList />}` and make it render the bookmarks all the time because the view will be bookmarks